tests/integration-tests/tests/common/assertions.py

```diff
@@ -52,6 +52,8 @@ def assert_no_errors_in_logs(remote_command_executor, scheduler):
 
     for log_file in log_files:
         log = remote_command_executor.run_remote_command("sudo cat {0}".format(log_file), hide=True).stdout
+        if skip_ice:
+            log = "\n".join([line for line in log.splitlines() if "InsufficientInstanceCapacity" not in line])
```

Comment:
NIT: Is "InsufficientInstanceCapacity" enough to catch **only** the ICE errors?
I'm wondering if we may end up missing errors that have the word "InsufficientInstanceCapacity" but are not actually ICE errors.